packages/enzyme-adapter-react-16.1/src/ReactSixteenOneAdapter.js

```diff
@@ -295,8 +295,12 @@ class ReactSixteenOneAdapter extends EnzymeAdapter {
         return instance ? toTree(instance._reactInternalFiber).rendered : null;
       },
       simulateError(nodeHierarchy, rootNode, error) {
+        const isErrorBoundary = el => (
+          el.componentDidCatch || (el.constructor && el.constructor.getDerivedStateFromError)
```

Comment:
Since the argument isn't an element, it's an instance, let's call it `instance`.